### PR TITLE
Fix bug 1134522 - New leadership page

### PR DESCRIPTION
**Pull request by craigcook**
DO NOT MERGE

Still a few bits of content I'm waiting to finalize but the code is complete enough to start review.

@flodolo: This is a ton of new strings, but I think we shouldn't expose them to localizers for a while, let things settle down after MWC at least. The only thing we may want to expose soon is the new bit on the main /about page, which I've wrapped in an l10n tag, but that text isn't final.

### CODE REVIEW

**Comment by flodolo** [comment]
(cc @pascalchevrel) We have a system to manage opt-in pages now.
https://l10n.mozilla-community.org/~pascalc/langchecker/?action=optin&locale=da

While we need the strings in about-base and about for all locales (with the next batch of updates after MWC), I think /leadership is a perfect candidate for the opt-in group.

I wonder if we need the tag in the navigation, or we just need to live with it.

File: bedrock/mozorg/templates/mozorg/about/leadership.html [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+    <ul class="gallery">
+      <li class="vcard" itemscope itemtype="http://schema.org/Person">
+        <a class="url" itemprop="url" href="https://blog.mozilla.org/press/bios/mitchell-baker/">
+          <img class="photo" itemprop="image" alt="" src="{{ static('img/mozorg/about/leadership/mitchell-baker.jpg') }}">
+          {# L10n: This "Chair" is the chairperson of the board of directors, not something to sit on #}
+          <span class="fn" itemprop="name">Mitchell Baker, {{ _('Chair') }}</span>
+        </a>
+      </li>
+      <li class="vcard" itemscope itemtype="http://schema.org/Person">
+        <a class="url" itemprop="url" href="https://blog.mozilla.org/press/bios/mitchell-baker/">
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about/leadership.html]: This link should be `https://blog.mozilla.org/press/bios/chris-beard/`

**Comment by alexgibson** [comment]
Aside from the one incorrect link, this all looks good to me so far :+1: 

Also tested the markup using https://developers.google.com/structured-data/testing-tool/ which gives 37 valid hcards/Persons.

File: bedrock/mozorg/templates/mozorg/about/leadership.html [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+      <li><a href="#management">{{ _('Management Teams') }}</a></li>
+      <li><a href="#reps-council">{{ _('Mozilla Reps Council') }}</a></li>
+      <li><a href="#directors">{{ _('Boards of Directors') }}</a></li>
+    </ul>
+  </nav>
+
+  <section id="management" class="brass">
+    <h2>{{ _('Management Teams') }}</h2>
+
+    <h3 id="mgmt-corp"><a href="{{ url('foundation.moco') }}">{{ _('Mozilla Corporation') }}</a></h3>
```
Comment (kyoshino) [inline bedrock/mozorg/templates/mozorg/about/leadership.html]: (maybe 5) http://schema.org/Organization and the `member` properties for each person can be used here and there. Not a release blocker though.

**Comment by jpetto** [comment]
Gave it a good once over and it looks ready to me. r+

File: bedrock/mozorg/templates/mozorg/about/leadership.html (hunk 2 above)
Comment (craigcook) [inline bedrock/mozorg/templates/mozorg/about/leadership.html]: Not a bad idea but I don't want to make any more changes now. I might add organization stuff next week.

**Pull request merged**